contracts/priv/fluxdocsd.yml

```diff
@@ -147,6 +147,7 @@ components:
         - kind
         - headline
         - fluxType
+        - example
```

Comment:
I don't think we need this on the backend. Let's make it optional rather than required by removing it from the required list.